doc/faq_use_gpu_on_slurm_cluster.md

```diff
@@ -81,6 +82,14 @@ shown below.
    srun ./engine run --single-job {{job.fw_id}}
 ```
 
+### Compute Engine
+
+Ensure that you have a Compute Engine installed that has been compiled after 2024-02-01.
+Please contact Flywheel staff to get an updated Flywheel engine.
+
+After receiving the updated Flywheel engine, install it as per the instructions found
+[here](faq_updating_flywheel_engine.md).
+
 ## Gear Execution
 
 With the rest of the workflow configured, adding a `gpu` tag (in addition to the
```

Comment:
Somewhere here it should have:

"Note: if your site already uses the `gpu` tag for launching another engine on Flywheel, contact Flywheel staff if your job is picked up by this engine instead of the HPC Hold engine (and HPC Client)."